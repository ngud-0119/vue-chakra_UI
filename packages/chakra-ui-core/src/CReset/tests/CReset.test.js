import CReset from '../'
import { render, defaultProviders } from '@/tests/test-utils'

const renderComponent = (options = {}) => {
    return render(
        CReset, 
        options, 
        () => ({ provide: defaultProviders() })
    )
}

describe('CReset.vue', () => {

  it('should render correctly', () => {
    const { asFragment } = renderComponent()

    expect(asFragment()).toMatchSnapshot()
  })

})
