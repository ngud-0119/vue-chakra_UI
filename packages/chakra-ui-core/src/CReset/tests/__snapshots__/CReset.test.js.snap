// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CReset.vue should render correctly 1`] = `
<DocumentFragment>
  <!---->
</DocumentFragment>
`;
